bycycle/cyclepoints/extrema.py

```diff
@@ -51,13 +54,20 @@ def find_extrema(sig, fs, f_range, boundary=None, first_extrema='peak', filter_k
     if filter_kwargs is None:
         filter_kwargs = {}
 
-    # Default boundary value as 1 cycle length of low cutoff frequency
-    if boundary is None:
-        boundary = int(np.ceil(fs / float(f_range[0])))
+    # Pad beginning of signal with zeros to prevent missing cyclepoints
+    if pad:
```

Comment:
This is minor / but I think this could be condensed a little, by doing the padding within this if. 
```
if pad:
    filt_len = compute_filter_length(...)
    sig = pass # line to pad the signal, without renaming it
```

One reason: we currently rename the variable to `sig_pad`, but it's actually not always padded. I think keeping it as `sig` is fine / makes more sense (we don't need a rename).

Reply:
Agreed, condensed and the name is kept as `sig`. I also simplified using `sig = np.pad(sig, int(filt_len/2))`.